tornado-unittests/src/main/java/uk/ac/manchester/tornado/unittests/multithreaded/TestMultiThreadedExecutionPlans.java

```diff
@@ -164,7 +164,7 @@ public void test03() {
         for (int i = 0; i < 100; i++) {
             int finalI = i;
             Thread t1 = new Thread(() -> compute(1014 * 1024, finalI, false));
-            Thread t2 = new Thread(() -> compute(1014 * 1024, finalI, false));
+            Thread t2 = new Thread(() -> compute(1014 * 1024, finalI + 100, false));
```

Comment:
Is this summation required for this particular test? Is the point to give a different value from t1?

Reply:
This parameter in this test is used for setting the name of the Task-Graph. We need unique names, unless we reuse code for different execution plans. But this test is intended to check different execution plans with different input task-graphs to mimic the scenario of one of the GAIA use cases.